src/hooks/constants/resolvers/query/constants/constants.tsx

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import { ApiPromise } from '@polkadot/api';
+import { Constants } from '../../../../../generated/graphql';
+import { usePolkadotJsContext } from '../../../../polkadotJs/usePolkadotJs';
+import { withErrorHandler } from '../../../../apollo/withErrorHandler';
+import errors from '../../../../../errors';
+
+export const __typename: Constants['__typename'] = 'Constants';
+const withTypename = () => ({
+  __typename,
+});
+
+/**
+ * Resolver for the `Constants` entity
+ *
+ * @param apiInstance ApiPromise
+ */
+export const constantsQueryResolverFactory =
+  (apiInstance?: ApiPromise) => () => {
+    if (!apiInstance) throw Error(errors.apiInstanceNotInitialized);
```

Comment:
whats the point for `constants` or `LBPConstants` to pass the `apiInstance` at all? since its just an empty resolver to provide the `typename` for the nested resolver structure?

Reply:
fixed